packages/@lwc/compiler/package.json

```diff
@@ -23,16 +23,16 @@
         "dist/"
     ],
     "dependencies": {
-        "@babel/core": "~7.11.6",
-        "@babel/plugin-proposal-class-properties": "~7.12.1",
-        "@babel/plugin-proposal-object-rest-spread": "~7.12.1",
+        "@babel/core": "7.13.8",
```

Comment:
Hardcoding the babel version doesn't do anything since all the transitive dependencies are versioned using `^`. This approach only pins the top-level babel version but not the transitive one.